boa_interner/src/lib.rs

```diff
@@ -297,9 +298,15 @@ impl Sym {
     }
 
     /// Retrieves the raw `NonZeroUsize` for this symbol.
+    #[cfg(not(feature = "fuzzer"))]
     const fn as_raw(self) -> NonZeroUsize {
         self.value
     }
+
+    #[cfg(feature = "fuzzer")]
+    pub const fn as_raw(self) -> NonZeroUsize {
+        self.value
+    }
```

Comment:
If users just need to use a feature to call a private function, I'd just expose it as public in the first place. It doesn't really matter in this case since getting a raw `NonZeroUsize` is not useful to interact with the internet.

Reply:
This change comes from a place of "change as little as possible of the underlying implementation". :) I'll just replace it.